### PR TITLE
grammar pass

### PR DESCRIPTION
**Pull request by krzychukula**
I didn't want to touch the quote:

```
> We have lost our ability to achieve more with less. We do more with more. Or, in certain cases, we do less with more.
```

- [ ] it misses an author (and couldn't find it with google)
- [ ] I could suggest something like:

```
> We have lost our ability to achieve more with less. We do more with more. Or, sometimes, we do less with more.
```

### CODE REVIEW

File: manuscript/ch1.md [hunk 1]
```diff
@@ -46,20 +46,20 @@ By the end of this book you will learn to:
...
 
 You will find a deployed version of this app on Netlify: https://hyperposts.netlify.app/
 
 You will find source code on Github: https://github.com/kwasniew/hyperbook-tutorial
 
-Throughout the book you'll be building 2 application screens:
+Throughout the book, you'll be building two application screens:
 
-* main screen with a post submit form and a list of posts streamed from the server
+* main screen with a new post form and a list of posts streamed from the server
```
Comment (krzychukula) [inline manuscript/ch1.md:62]: This is the biggest change. `post submit form` is unclear to me.

File: manuscript/ch1.md [hunk 2]
```diff
@@ -46,20 +46,20 @@ By the end of this book you will learn to:
...
 You will be building a messaging/chat application called **HyperPosts**. I decided to choose a boring and familiar domain purposefully. 
-To ease the learning process you will only focus on the framework concepts instead of learning new and unfamiliar domain.
+To ease the learning process, you will only focus on the framework concepts instead of learning new and unfamiliar domain.
 
 You will find a deployed version of this app on Netlify: https://hyperposts.netlify.app/
 
 You will find source code on Github: https://github.com/kwasniew/hyperbook-tutorial
 
-Throughout the book you'll be building 2 application screens:
+Throughout the book, you'll be building two application screens:
```
Comment (krzychukula) [inline manuscript/ch1.md:60]: I'm not 100% sure about this change. Grammar tools flag it by I personally prefer numbers. What would you prefer?

Comment (kwasniew) [reply manuscript/ch1.md:60]: Let's go with the suggestion. As long as we stay consistent throughout the text and the numbers are small.

**Comment by kwasniew** [comment]
The quote was inspired by this tweet (https://twitter.com/astralwave/status/1258918850625708032) but I paraphrased it.

**Pull request merged**